packages/binding-coap/test/coap-client-test.ts

```diff
@@ -88,7 +86,7 @@ class CoapClientTest {
         await coapServer.stop();
     }
 
-    @test "should re-use port"(done: Function) {
+    @test "should re-use port"(done: () => {}) {
```

Comment:
There's a type in mocha for the done function, I think is `Mocha.Done`

Reply:
Done ✅